build-system/tasks/check-types.js

```diff
@@ -40,11 +40,41 @@ const PRIDE_FILES_GLOBS = [
   // Core
   'src/core/**/*.js',
 
+  // Polyfills
+  'src/polyfills/abort-controller.js',
```

Comment:
Are polyfills required to clear something that depends on them? If not, I'd say leave them out for now; I have a branch that's about to be PR'd that moves them all into core

Reply:
Removed